src/ui/spaceholder/Spaceholder.tsx

```diff
@@ -1,17 +1,20 @@
-import React, { memo } from 'react';
+import React, { memo, ReactNode } from 'react';
 
 import csx from './Spaceholder.scss';
 
 namespace Spaceholder {
   export interface Props {
     className?: string;
     theme?: 'primary';
+    children?: ReactNode;
   }
 }
 
 const Spaceholder = memo(
-  ({ className, theme = 'primary' }: Spaceholder.Props) => (
-    <div className={`${csx.spaceholder} ${className} ${csx[theme]}`} />
+  ({ className, theme = 'primary', children }: Spaceholder.Props) => (
```

Comment:
classsName = ''" instead we will have undefined in markup

Reply:
done